### ISSUE TITLE
Can which-key detect only `Ctrl` is pressed?

### ISSUE THREAD

**Issue by avatar-lavventura** [justbur/emacs-which-key#251]
I could not not following that when I press only ctrl, can which-key able to show its bindings such as `C-n`, `C-p` and so on.

**Comment by justbur** [comment]
No, unfortunately not. Control by itself does not count as an input event for emacs, and which-key pops up after a incomplete sequence of input events (incomplete key sequence).

**Comment by avatar-lavventura** [comment]
@justbur Thanks for the input
